src/config/config_def.c

```diff
@@ -178,6 +178,11 @@ static const WT_CONFIG_CHECK confchk_WT_CONNECTION_reconfigure[] = {
 	{ NULL, NULL, NULL, NULL, NULL, 0 }
 };
 
+static const WT_CONFIG_CHECK confchk_WT_CONNECTION_rollback_to_stable[] = {
+	{ "timestamp", "string", NULL, NULL, NULL, 0 },
+	{ NULL, NULL, NULL, NULL, NULL, 0 }
+};
```

Comment:
I could go with @sueloverso's suggestion, or alternatively stick with `rollback_to_stable`, but don't have the application pass a timestamp at all -- i.e., always roll back to the last `stable_timestamp` that was set (and fail if none was ever set).

The global `stable_timestamp` is unavoidable (we need it for checkpoints and for eviction) -- I don't know why we need to pass a (potentially different) timestamp here...